src/sass/_fixes.scss

```diff
@@ -0,0 +1,17 @@
+@mixin fixText($fontSize, $lineHeightOffset: 0) {
+  // $lineHeightOffset is a way to fix font issue with the baseline
+  // the issue is visible when vertically centering the text of at least 12px
+  font-size: $fontSize;
+  height: $fontSize;
+  line-height: $fontSize + $lineHeightOffset;
+}
+
+@mixin fixOperaMiniLabelText {
```

Comment:
maybe we should use Opera Mini specific selector like:
`:-o-prefocus, .block1, .block2 {font-size: 14px;}` ?